src/device.rs

```diff
@@ -504,7 +504,37 @@ pub unsafe extern "C" fn wgpuDeviceCreateRenderPipeline(
             polygon_mode: wgt::PolygonMode::Fill,
             conservative: false,
         },
-        depth_stencil: descriptor.depthStencil.as_ref().map(|_| unimplemented!()),
+        depth_stencil: descriptor
+            .depthStencil
+            .as_ref()
+            .map(|desc| wgt::DepthStencilState {
+                format: map_texture_format(desc.format)
+                    .expect("Texture format must be defined in DepthStencilState"),
+                depth_write_enabled: desc.depthWriteEnabled,
+                depth_compare: map_compare_function(desc.depthCompare).unwrap(),
+                stencil: wgt::StencilState {
+                    front: wgt::StencilFaceState {
```

Comment:
can we have a `map_stencil_face` helper to avoid repeating ourselves?

Reply:
Yes!